conf.yaml

```diff
@@ -1768,8 +1768,8 @@ contents:
           - title:      Ingest Management Guide
             prefix:     en/ingest-management
             current:    *stackcurrent
-            branches:   [ master, 7.x, 7.9, 7.8 ]
-            live:       [ master, 7.x, 7.9, 7.8 ]
+            branches:   [ master, 7.x, 7.10, 7.9, 7.8 ]
+            live:       [ master, 7.x, 7.10, 7.9, 7.8 ]
```

Comment:
Should we take 7.8 out of this list? Or can we change it to `*stacklive`?

Reply:
I think this is a question for @dedemorton.  Can happen separately from this PR (before or after) as far as I'm concerned.

Reply:
@lcawl I think I did that before because I didn't have all the branches in the list. You should be able to use *stacklive now.